tests/system/fim/common_tasks/delete_files.py

```diff
@@ -12,8 +12,25 @@
 import logging
 import os
 import time
-
-def delete_files(input_file_path, n, output_file_path, bunch_size=100, wait_time=100):
+def delete_file(path, attempt=0, sleep_time=5):
+    success = True
+    try:
+        if attempt > 0:
+          time.sleep(sleep_time)
+          logging.info(f"Failed to delete {path}, retry: {attempt}")
```

Comment:
Just noting that F-strings are only available since Python 3.6, not sure if there's an environment with 3.5 or older, otherwise just ignore this comment.

Reply:
Yes, I know it, I assume we'll have at least python 3.5